docs/examples/pandapower_example.ipynb

```diff
@@ -513,11 +704,11 @@
       ],
       "text/plain": [
        "        vm_pu  va_degree      p_mw    q_mvar\n",
-       "101  1.000000   0.000000 -1.798661 -3.476701\n",
-       "102  0.973746 -30.017299  0.000000  0.000000\n",
-       "103  0.973014 -30.007075  2.414572  0.231799\n",
-       "104  0.969519 -59.886045  0.281990  1.550945\n",
-       "105  0.972063 -59.860441 -0.944398 -0.581069\n",
+       "101  1.000000   0.000000 -1.877113 -3.640600\n",
+       "102  0.993171 -30.016181  0.000000  0.000000\n",
+       "103  0.992419 -30.005563  2.475814  0.237678\n",
+       "104  0.988794 -59.887701  0.293314  1.613227\n",
+       "105  0.991372 -59.864589 -0.940607 -0.577278\n",
```

Comment:
someone with EE knowledge should review these values. Possibly related to the pandapower 3.x difference in modelling